src/ucp/core/ucp_context.h

```diff
@@ -260,6 +260,21 @@ typedef struct ucp_tl_iface_atomic_flags {
     };
 
 
+/**
+ * Check if at least one feature flag from @a _flags is initialized.
+ */
+#define UCP_CONTEXT_CHECK_FEATURE_FLAGS(_context, _flags, _status, _action) \
+    do { \
+        if (ENABLE_PARAMS_CHECK && !((_context)->config.features & (_flags))) { \
+            char feature_list_str[512] = {0}; \
+            ucs_error("required feature flags \"%s\" are not enabled", \
```

Comment:
1. Do you want to say: "are not support by underlaying transport"
2. If this is the case it might be helpful to print out the list of transports that have been used.

Reply:
No, the case is the used forgot to request required features by ucp_init

Reply:
"the feature can be enabled using XXX call"